core/src/main/java/org/apache/accumulo/core/file/rfile/bcfile/Compression.java

```diff
@@ -456,6 +458,122 @@ public boolean isSupported() {
 
         return snappyCodec != null;
       }
+    },
+
+    ZSTANDARD(COMPRESSION_ZSTD) {
```

Comment:
This is custom code, copied from our other enums.  The Hadoop Compression.java[1] that this was originally copied from has not changed (it only has lzo and gz). I thought of refactoring this class since we have branched off pretty far but that could be a follow on.
[1] https://github.com/apache/hadoop/blob/trunk/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/file/tfile/Compression.java

Reply:
Oh, interesting. I assumed Hadoop supported snappy already.

Reply:
They do support snappy and zstandard but somewhere else in the code I guess?